lenses/tmpfiles.aug

```diff
@@ -50,15 +50,16 @@ Empty lines *)
 
   (* View: type
 One letter. Some of them can have a "+" and all can have an
-exclamation mark ("!") and/or minus sign ("-").
+exclamation mark ("!"), a minus sign ("-"), an equal sign ("="),
+a tilde character ("~") and/or a caret ("^").
 
 Not all letters are valid.
 *)
-  let type     = /([fFwdDevqQpLcbCxXrRzZtThHaAm]|[fFwpLcbaA]\+)!?-?/
+  let type     = /([fFwdDevqQpLcbCxXrRzZtThHaAm]|[fFwpLcbaA]\+)!?-?=?~?\^?/
```

Comment:
The expression
  `!?-?=?~?\^?`
makes each of these characters optional on its own, but when used in combination, the expression implies a specific order that is not mentioned in the documentation.
eg: the expression would accept `~^` but not `^~`

It would be better to use a more loose expression like this:
 `[-!=~^]*`

I'm not sure if this example would need parenthesis `([-!=~^]*)`

Reply:
makes sense, even the documentation says that there can be more than one -- changed, thanks!